stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatClientConfig.kt

```diff
@@ -1,24 +1,17 @@
 package io.getstream.chat.android.client.api
 
 import io.getstream.chat.android.client.logger.ChatLogger
-import io.getstream.chat.android.client.notifications.handler.ChatNotificationHandler
-import io.getstream.chat.android.client.token.TokenManager
-import io.getstream.chat.android.client.token.TokenManagerImpl
-import io.getstream.chat.android.client.uploader.FileUploader
 
-internal class ChatClientConfig(
-    val apiKey: String,
-    var httpUrl: String,
-    var cdnHttpUrl: String,
-    var wssUrl: String,
-    var baseTimeout: Long,
-    var cdnTimeout: Long,
-    val warmUp: Boolean,
-    val loggerConfig: ChatLogger.Config,
-    val notificationsHandler: ChatNotificationHandler,
-    val fileUploader: FileUploader? = null,
-    val tokenManager: TokenManager = TokenManagerImpl()
+public class ChatClientConfig(
+    public val apiKey: String,
+    public var httpUrl: String,
```

Comment:
I know that you're not the creator of this class, but perhaps this can be changed to val instead of var.